docs/all-rules.md

```diff
@@ -3,10 +3,10 @@ id: all-rules
 title: All Rules
 ---
 
-- 🔧 - Meaning the rule can fix problems automatically by running eslint `--fix` options.
+- 🔧 - Meaning the rule can fix problems aotomatically by running eslint `--fix` options.
 - ⭐ - Meaning the rule is recommended.
 
-### Best Practice
+### Best Pracice
```

Comment:
The previous spelling is correct.

Reply:
@polyzen  Thanks 👍  It's fixed https://github.com/yeonjuan/html-eslint/pull/104/commits/bd35155cde5a0b9f2ab6d1e0890c4b8f29e70e22